src/main/webapp/index.html

```diff
@@ -483,13 +483,6 @@
             <polygon
                 points="298.667,30.187 268.48,0 149.333,119.147 30.187,0 0,30.187 119.147,149.333 0,268.48 30.187,298.667 149.333,179.52 268.48,298.667 298.667,268.48 179.52,149.333 " />
         </symbol>
-        <symbol id="upload-logo" viewBox="0 0 100 100" style="width:100%;height:100%;background-size:initial;background-repeat-y:initial;background-repeat-x:initial;background-position-y:initial;background-position-x:initial;background-origin:initial;background-image:initial;background-color:#fff;background-clip:initial;background-attachment:initial;animation-play-state:paused" xmlns="http://www.w3.org/2000/svg">
```

Comment:
looks like u have removed logo which is used for client logo in admin...pls add it back